citi-onboarding/maracatronics
# Feature/social media

## What i did
- Created the social media model;
- Created the social media controller;
- Created the social media route;

## How to test
1. First, you need to use the command ```git fetch --all```
2. Change your branch to "feature/social-media" with ```git checkout feature/social-media```
3. Now, you need to found the server folder with ```cd server```
4. On the server folder, run the server with ```node index.js``` and go to http://localhost:3001/admin
5. Go to "Redes sociais" model and edit the texts
6. Now, go to the route: http://localhost:3001/admin/api/redesSociais
7. On that route, check if the texts changed
